buildstockbatch/gcp/gcp.py

```diff
@@ -369,6 +368,8 @@ def run_batch(self):
             - package and upload the assets, including weather
             - kick off a batch simulation on GCP
         """
+        gcp_cfg = self.cfg['gcp']
```

Comment:
How do you decide what to make available as a property (during init) and what's fetched on-demand? I'm not seeing what the pattern is. Should probably add a comment/doc for others to know/follow.

Reply:
Honestly, there isn't a clear rule. Most things just follow what the AWS code does (which is a mix of saving props and reading directly from the config). I decided not to save these as properties because they're only used in one place, and they're used in code that we won't refactor to share with AWS.